projects/serializers.py

```diff
@@ -85,10 +84,39 @@ class ProjectControlSerializer(serializers.ModelSerializer):
     project = BasicViewProjectSerializer(read_only=True)
     catalog_data = serializers.SerializerMethodField(read_only=True)
     component_data = serializers.SerializerMethodField(read_only=True)
+    disable_narratives = serializers.ListSerializer(child=serializers.IntegerField(), write_only=True, required=False)
+    enable_narratives = serializers.ListSerializer(child=serializers.IntegerField(), write_only=True, required=False)
```

Comment:
@jauschalley New fields `disable_narratives` and `enable_narratives`. Should be a list, but if only updating one thing at a time it can just be a list of one.

Reply:
patch example:
```json
{
  "disable_narratives": [1, 2, 3],
  "enable_narratives": [4]
}
```